RevenueCatUI/Templates/V2/PaywallsV2View.swift

```diff
@@ -56,12 +57,17 @@ struct PaywallsV2View: View {
             wrappedValue: .init(introEligibilityChecker: introEligibilityChecker)
         )
 
+        // Step 0: Decide which ComponentsConfig to use (base is default)
+        let componentsConfig = paywallComponentsData.componentsConfig.base
+
         // The creation of the paywall view components can be intensive and should only be executed once.
         // The instantiation of the PaywallStateManager needs to stay in the init of the wrappedValue
         // because StateObject init is an autoclosure that will only get executed once.
         self._paywallStateManager = .init(
             wrappedValue: .init(state: Self.createPaywallState(
-                paywallComponentsData: paywallComponentsData,
+                componentsConfig: componentsConfig,
+                componentsLocalizations: paywallComponentsData.componentsLocalizations,
+                defaultLocale: paywallComponentsData.defaultLocale,
```

Comment:
Some reworks here to split the mega object that was going into the paywall state.

The `PaywallComponentsConfig` (aka `paywallComponentsData.componentsConfig.base` in this case) needed to be selected one and then used for the paywall background image below 👇